dokuwiki-text/src/main/java/org/xwiki/contrib/dokuwiki/text/internal/input/DokuWikiInputFilterStream.java

```diff
@@ -300,10 +302,14 @@ private void readDocument(File directory, String dokuwikiDataDirectory, DokuWiki
                             //read revisions
                             readPageRevision(file, dokuwikiDataDirectory, proxyFilter);
                         } else {
-                            documentLocaleParameters = readDocument(file,documentLocaleParameters, dokuwikiDataDirectory, proxyFilter);
+                            documentLocaleParameters = readDocument(file,documentLocaleParameters,
+                                    dokuwikiDataDirectory, proxyFilter);
                         }
-                    } catch (Exception e ){
-                        documentLocaleParameters = readDocument(file,documentLocaleParameters, dokuwikiDataDirectory, proxyFilter);
+                    }
+                    else {
+                        this.logger.warn("File" + f + "not found (Ignoring)");
```

Comment:
Would be better to give a short explanation of the impact this missing file can have instead of "Ignoring".